### ISSUE TITLE
Add InChi and SMARTS strings to the species_database.yml file

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#2357]
### Your name

Bob Yantosca

### Your affiliation

Harvard + GCST

### Provide a clear and concise overview of the new feature requested.

Jessica Haskins (@jhaskinsPhD) has compiled a list of International Chemical Identifier (InChi)) and SMARTS chemical identifiers for each of the GEOS-Chem species.  

We propose to add these as fields to the GEOS-Chem species database file (`species_database.yml`) as an additional reference for each species.  The SMARTS string in particular can describe the isomer structure of each molecule.

### Reference(s)

1. S. Heller et al, *InChI, the IUPAC International Chemical Identifier,* Journal of Cheminformatics, vol 7, article 23 (2015). [link](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-015-0068-4)
2. SMARTS: A language for describing molecular patterns [link](https://ics.uci.edu/~dock/manuals/DaylightTheoryManual/theory.smarts.html#:~:text=SMARTS%20is%20a%20language%20that,to%20those%20acquainted%20with%20SMILES.)

### Will you be implementing this feature yourself?

Yes, but I will need some help

**Comment by jhaskinsPhD** [comment]
Yes. I have almost completed it... I have all the explicit species mapped, & am playing around with a few ideas on the lumped compounds. Will report back shortly.
